app/src/main/java/com/tpp/theperiodpurse/ui/symptomlog/LogScreen.kt

```diff
@@ -220,14 +224,14 @@ fun LogScreenLayout(
 
 @RequiresApi(Build.VERSION_CODES.O)
 @Composable
-fun LogScreenTopBar(navController: NavController, date: LocalDate) {
-    PopupTopBar(onClose = { navController.navigateUp() }) {
-        LogScreenTopBarContent(navController = navController, date = date)
+fun LogScreenTopBar(navController: NavController, date: LocalDate, appViewModel: AppViewModel) {
+    PopupTopBar(onClose = { navController.navigateUp() }, appViewModel = appViewModel) {
+        LogScreenTopBarContent(navController = navController, date = date, appViewModel = appViewModel)
     }
 }
 
 @Composable
-private fun LogScreenTopBarContent(navController: NavController, date: LocalDate) {
+private fun LogScreenTopBarContent(navController: NavController, date: LocalDate, appViewModel: AppViewModel) {
```

Comment:
The background color is too light compared to the design. It should be 0xFFA1A2A6